test/test_dag.py

```diff
@@ -322,8 +313,7 @@ def test_auxiliary_steps(clear_now_and_after):
     # a.txt exists and b.txt does not exist
     with open('a.txt', 'w') as atfile:
```

Comment:
This one is a `temp_factory('a.txt')`.

Reply:
Is there a way to replace the lines
```
    with open('test.dot') as dot:
        lc = len(dot.readlines())
```
with `temp_factory`?